Makefile

```diff
@@ -72,15 +72,17 @@ format-python: | $(VENV)
 	; done
 
 generate-api-docs:
-	npx openapi-generator generate -i ./templates/api.definition.yml -g markdown -t ./docs/templates/ -o docs/api
+	cat ./templates/api.yaml | $(VENV)/bin/yq -y .Resources.Api.Properties.DefinitionBody > ./templates/temp.definition.yml
```

Comment:
Nitpick: you should use mktemp when a temporary filename is needed - otherwise this could overwrite/delete an existing file.

Reply:
Thanks for the advice, i have done as suggested